docs/options.md

```diff
@@ -0,0 +1,30 @@
+# Changing PostgreSQL Options
+
+You may require a configuration change for your application. PostgreSQL
+allows the option to configure the database with a configuration file, as many other database
+management systems do. You can pass options to PostgreSQL instances using the
+specific Custom Resource option via the `deploy/cr.yaml` configuration file.
+
+Often there's no need to add custom options, as the Operator takes care of
+providing PostgreSQL with reasonable defaults.
+
+The `patroni` section in the Custom Resource, present in `deploy/cr.yaml` file,
+contains configuration options to customize the PostgreSQL high-availability
+implementation based on [Patroni](https://patroni.readthedocs.io/).
```

Comment:
```suggestion
The `patroni` section in the Custom Resource, present in `deploy/cr.yaml` file,
contains configuration options to customize the PostgreSQL.
```
Why do we want to mention patroni and so on? Who cares here that it is a solution for HA for PostgreSQL? 
The goal here is to explain how to change PG config through the operator.

Reply:
I would also make sure that we put the full variable name in the doc: `patroni.dynamicConfiguration.postgresql.parameters`.